src/Homeworks/Homework2_Task1/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Stack_Calculator;
+Console.WriteLine("Input polish postfix expression to calculate (elements must be separated by whitespace)");
+var Expression = Console.ReadLine();
+if (Expression == null) {
+    Console.WriteLine("Empty expression");
+    return;
+}
+var AStack = new StackArray();
```

Comment:
```suggestion
var arrayStack = new StackArray();
```
Лучше не сокращать.
Также файл, как и остальные, требует рефакторинга с помощью StyleCop